### DISCUSSION TITLE
Extension not working in CU

### DISCUSSION THREAD

**Discussion by akasaka99** [Eloston/ungoogled-chromium#1578] [Q&A]
Hi-
I'm new to CU so I am not sure where to report an issue so I am posting it here in the general discussion and maybe the user could tell me where to post it:
1. I am new to CU and just installed latest build 92 on my Mac
2. Installed a bunch of extension but one seems not to work: Redirector 3.5.3
(https://chrome.google.com/webstore/detail/redirector/ocgpenflpmgnfapjedencafcfakcekcd?hl=en)
=> Redirector automatically redirect content based on user-defined rules.
=> So for instance if you dont want to be tracked on by youtube you just use simple wildcard on Redirector and have it go to an Individious instance.
3. The extension works perfectly on Firefox, Chrome, Brave, etc except on CU... nothing is happening i.e. no redirection

So I wonder if it something in CU which on purpose does not allow this feature with the extension or a CU bug?
Maybe there's a more appropriate extension equivalent to Redicrect to CU?
Thanks.

**Comment by wchen342** [comment]
It will be hard to tell without further information. Try open developer tools with F12 and go to Console, then refresh. Does and error shows up?
